dev/PushNotifications/GetRawNotificationEventArgs.h

```diff
@@ -10,8 +10,19 @@ constexpr PCWSTR c_pushContractId = L"Windows.Push";
 
 namespace winrt::Microsoft::Windows::PushNotifications
 {
-    static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const&)
+    static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& uri)
     {
+        // Verify if the uri contains a background notification payload.
+        // Otherwise, we expect to process the notification in a background task.
+        std::wstring uriAsString{ uri.AbsoluteUri() };
+        if (uriAsString._Starts_with(L"-payload:"))
```

Comment:
I think this can be simplified if the command line payload were URI compliant from the beginning. I'll try my best to get a better solution.